index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Instruments</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <header>Music</header>
+      <a href="#main_types_of_instruments" class="nav-link" id="main-types">Main Types of Instruments</a>
```

Comment:
As a kindly suggestion, please remember we use "h" tags for titles, we use headers for the introductory content of the website, like a section tag but for the introduction.
example:
`<h1>Music</h1>`
Currently, all your titles are using the header tag instead of the h tags, almost there, just a little bit more 🙌🏼

Reply:
> As a kindly suggestion, please remember we use "h" tags for titles, we use headers for the introductory content of the website, like a section tag but for the introduction. example: `<h1>Music</h1>` Currently, all your titles are using the header tag instead of the h tags, almost there, just a little bit more 🙌🏼

- [ ] Kindly consider to make sure your comments are clear and easy to understand. For Example:

This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

- [ ] I would kindly suggest you to always includes an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*